src/routes/cluster_workload_info.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+"""Contains the handlers and business logic for the workload API."""
+from collections import defaultdict
+
+import marshmallow
+from connexion.lifecycle import ConnexionResponse
+
+from src.schemas.cluster_workload_info_schema import (
+    ClusterWorkloadInfoSchema,
+    AccountWorkloadResponseSchema,
+    AccountsWorkloadRequestSchema,
+)
+from src.models.cluster_workload_info_model import ClusterWorkloadInfoModel
+
+
+def get_workload_info(body):
+    """Another stub to ensure connexion does not error out."""
+    try:
+        body = AccountsWorkloadRequestSchema().load(body)
+    except marshmallow.exceptions.ValidationError:
+        return "Invalid values supplied in JSON body, please review API limits", 400
+    workload_data = (
+        ClusterWorkloadInfoModel.query.filter(ClusterWorkloadInfoModel.account_id.in_(body.get("accounts", [])))
+        .order_by(ClusterWorkloadInfoModel.account_id, ClusterWorkloadInfoModel.cluster_id)
+        .offset(body.get("offset", 0))
+        .limit(body.get("record_count", 50))
```

Comment:
this limit will limit the number of records. that means it could include multiple clusters for an account. the total count seems to be counting the number of account records.

Reply:
`cls.query.filter(cls.account_id.in_(accounts)).count()` will translate to `SELECT count(*) FROM cluster_detail WHERE account_id IN (1, 2, 3..);`. It will have all the clusters of each of the accounts.

So the total record count will be the total across all pages, where a part of result is returned every time.